task/buildah-24gb/0.1/patch.yaml

```diff
@@ -7,3 +7,9 @@
 - op: replace
   path: /spec/steps/0/computeResources/requests/memory
   value: 20Gi
+- op: replace
+  path: /spec/steps/0/computeResources/limits/cpu
+  value: "10"
+- op: replace
+  path: /spec/steps/0/computeResources/requests/cpu
+  value: "20"
```

Comment:
I think you reversed the numbers, should be requests 10 and limits 20

Reply:
fixed